Mathlib/CategoryTheory/Sums/Basic.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kim Morrison, Robin Carlier
 -/
 import Mathlib.CategoryTheory.Equivalence
+import Mathlib.CategoryTheory.Products.Basic
```

Comment:
The new code should probably go in a separate file `CategoryTheory.Sums.Products`.